app/models/division_parameter_parser.rb

```diff
@@ -0,0 +1,48 @@
+class DivisionParameterParser
+
+  YEAR_ONLY_REGEX = /^\d{4}$/
+  YEAR_AND_MONTH_REGEX = /^\d{4}-\d{1,2}$/
+  COMPLETE_DATE_REGEX = /^\d{4}-\d{1,2}-\d{1,2}$/
+
+  def self.get_date_range(date)
+
+    if date =~ YEAR_ONLY_REGEX
+      return get_year_range(date)
```

Comment:
Actually, you are right. Here I can use implicit return. However, but in the `get_year_range` I can't because it returns more than 1 value.